package.json

```diff
@@ -2489,6 +2489,316 @@
 											}
 										]
 									},
+									{
+										"key": "languageId:latex",
+										"name": "LaTeX",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format region or buffer",
+														"type": "command",
+														"command": "editor.action.format"
+													},
+													{
+														"key": "b",
+														"name": "Format buffer",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "c",
+														"name": "Format changes",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													}
+												]
+											},
+											{
+												"key": "b",
+												"name": "+Backend",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "l",
+														"name": "View Workshop Messages",
+														"type": "command",
+														"command": "latex-workshop.log"
+													},
+													{
+														"key": "m",
+														"name": "Insert root magic comment",
```

Comment:
I am not sure what magic comment is but base on the name would that be more logically be in one of the following locations?

1. Insert (`SPC m i`)
2. Comment (`SPC m ;`)
3. Text (`SPC m x`)

Reply:
Anything that falls into the category of extension specific commands I've placed in `Backend`. The magic comment is one of those things (See https://github.com/James-Yu/LaTeX-Workshop/wiki/Compile#magic-comments).